wunderio/next4drupal-project
# Nex 75: Contacts list

[*Nex 75*](https://wunder.atlassian.net/browse/NEX-75)

*Changes proposed in this PR:*
- Created a contact person list section

*How to test:*
1. It's tricky with the type of image you may use, however, any portrait with some space around the head of the person should work fine. Play with the screen sizes to see if there are any breaks as well. 

*Link to feature environment:*
[Link](https://feature-nex-75-contacts-next.next4drupal-project.dev.wdr.io/)
